src/i18n.js

```diff
@@ -1,21 +1,20 @@
 import i18n from 'i18next';
 import { initReactI18next } from 'react-i18next';
 import LanguageDetector from 'i18next-browser-languagedetector';
-import TranslationsEn from './translations/english.json';
-import TranslationsSp from './translations/spanish.json';
-
-const resources = {
-  en: { translation: TranslationsEn },
-  es: { translation: TranslationsSp },
-};
+import Backend from 'i18next-http-backend';
 
 i18n
+  .use(Backend)
   .use(initReactI18next)
   .use(LanguageDetector)
   .init({
-    resources,
     supportedLngs: ['en', 'es'],
     fallbackLng: 'en',
+    backend: {
+      loadPath: `${process.env.PUBLIC_URL}/locales/{{lng}}/{{ns}}.json`,
+      addPath: `${process.env.PUBLIC_URL}/locales/add/{{lng}}/{{ns}}`,
+    },
+    ns: ['contact'],
```

Comment:
If we merge this as is, we'll lose all the translations except for the ones added in the `contact` namespace. Can we instead in this PR rename the existing translation files as part of a general/common namespace and load that everywhere we need it?